projects/landing-page/src/App.tsx

```diff
@@ -139,9 +140,29 @@ const App: React.FunctionComponent = () => {
               </SectionRef>
             </CardProject>
           </Section>
+          <Section>
+            <SectionHeading id='extension' prefix='6'>Extension</SectionHeading>
```

Comment:
I would keep extension in projects for now. We're not promoting it yet, not on v0. I imagine when we will promote it, we will need a dedicated page for it. For download link I would use same button as "Next Project" ProjectCard does